staging/src/k8s.io/client-go/tools/reference/ref.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package ref
+package reference
```

Comment:
To keep us on the same page: we still copy the clientset/listers/informers from kubernetes to client-go. I'm working on PRs to remove the copies in kubernetes.